CalibTracker/SiPixelLorentzAngle/python/SiPixelLorentzAnglePCLWorker_cfi.py

```diff
@@ -6,14 +6,7 @@
     folder = cms.string('AlCaReco/SiPixelLorentzAngle'),
     notInPCL = cms.bool(False),
     fileName = cms.string('testrun.root'),
-    newmodulelist = cms.vstring("BPix_BmI_SEC7_LYR2_LDR12F_MOD1",
-                                "BPix_BmI_SEC8_LYR2_LDR14F_MOD1",
-                                "BPix_BmO_SEC3_LYR2_LDR5F_MOD1",
-                                "BPix_BmO_SEC3_LYR2_LDR5F_MOD2",
-                                "BPix_BmO_SEC3_LYR2_LDR5F_MOD3",
-                                "BPix_BpO_SEC1_LYR2_LDR1F_MOD1",
-                                "BPix_BpO_SEC1_LYR2_LDR1F_MOD2",
-                                "BPix_BpO_SEC1_LYR2_LDR1F_MOD3"),
+    newmodulelist = cms.vstring(),
     src = cms.InputTag("TrackRefitter"),
     binsDepth    = cms.int32(50),
     binsDrift =    cms.int32(200),
```

Comment:
Perhaps you can clone SiiPixelLorentzAnglePCLWorker from the siPixelLorentzAnglePCLWorker generated by the fillDescription method, modifying only the parameters which have to be changer with respect to the default ones (here `binsDrift`, unless there is a typo somewhere)

Reply:
done at https://github.com/cms-sw/cmssw/pull/36565/commits/3b421bc3a014a8c63325bc4c6448220c2d073621